study.md

```diff
@@ -22,5 +22,17 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 In your own words, describe why someone would use Ember.
 
 ```md
-<!-- your answer here -->
+Ember allows for a unified way of dealing with user events. Instead of using Ajax to query your database for information, Ember has its own function that does the same thing with a much more concise and compact syntax.
```

Comment:
Ehhh... sort of. It's true that you're not using AJAX to write queries directly; that's because Ember is taking care of it behind the scenes. User events are something different, though.